front/src/Components/EmoteMenu/EmoteMenu.svelte

```diff
@@ -15,10 +16,31 @@
             rootElement: emojiContainer,
             styleProperties: {
                 "--font": "Press Start 2P",
+                "--text-color": "whitesmoke",
```

Comment:
What do you use `whitesmoke` color ?

Reply:
Whitesmoke is used on many parts of the project and the previous colour of the text does not have sufficient contrast to be seen by the colour blind.